### PR TITLE
Add new icons, labels, typeface, 

### PR DESCRIPTION
**Pull request by g4floyd**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,19 +2,26 @@
 
 Simple ES6 drum sequencer utilizing the [Web Audio API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Audio_API).
 
+Install all dependencies with `npm install`
+
+Make sure you have webpack-dev-server installed as well. You can install this with `npm install -g webpack-dev-server`
```
Comment (johnjoseph3) [inline README.md]: I added webpack-dev-server to the package.json file. This isn't necessary.

File: client/stylesheet.less [hunk 2]
```diff
@@ -55,12 +68,35 @@ body {
...
 
+.row-label-column {
+	flex: 0.5;
+	.row-labels {
+		display: flex;
+		flex-direction: column;
+		color: white;
+		height: 100%;
+		display: flex;
+		text-align: right;
```
Comment (johnjoseph3) [inline client/stylesheet.less]: You have two display properties in this block. Also, I don't think text-align is doing anything here.

File: client/drum-pad.html [hunk 3]
```diff
@@ -1,29 +1,38 @@
-<div ng-controller="drumPad">
+<div class="app-container" ng-controller="drumPad">
```
Comment (johnjoseph3) [inline client/drum-pad.html]: Generally, there is no reason to add a class unless you are going to use it to add css. It doesn't look like this class is being defined is css, so I would just remove it for now. If you're going for readability or browser indexing benefits, you want to do this type of thing: http://www.w3schools.com/html/html5_semantic_elements.asp.

**Review by johnjoseph3 — CHANGES_REQUESTED** [review]
A few comments. Small changes before merge.

File: client/drum-pad.html (hunk 3 above)
Comment (g4floyd) [reply client/drum-pad.html]: Makes sense. This was the beginnings of a next step. I'll remove it for this pull request.

**Comment by g4floyd** [comment]
Changes made. Let me know if there is anything else.

**Pull request merged**